c2corg_ui/templates/helpers/view.html

```diff
@@ -216,7 +224,11 @@ <h3 class="heading show-phone" ng-click="mainCtrl.animateHeaderIcon($event)" dat
           <app-card app-card-doc="doc"></app-card>
           % if type == 'articles':
             <app-delete-association child-doctype="articles" parent-id="${document['document_id']}" child-id="doc.document_id"
-              ng-if="userCtrl.auth.isAuthenticated()"></app-delete-association>
+              ng-if="userCtrl.auth.isAuthenticated()
+              % if isCollaborative:
```

Comment:
But your test seems a bit weird to me: if the document is collaborative, only moderators can remove the associations, but if the document is not collaborative (ie. personal), any authenticated user can remove the associations!?